nservicebus/upgrades/6to7/moving-msmq-subscriptions.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Migrating Msmq subscription messages
+reviewed: 2017-10-05
+component: MsmqTransport
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 6
+ - 7
+---
+
+When using Msmq Subscription storage in NServiceBus Versions 5.x and Versions 6.x, when a subscription queue is not configured, the subscription storage queue automatically defaulted to `NServiceBus.Subscriptions`. However, as this queue name was not endpoint specific, endpoints deployed to the same server could potentially share the same subscriptions queue which is undesirable. 
+
+To rectify this, starting from Versions 1.0 and above of NServiceBus.Transport.Msmq, if a queue called `NServiceBus.Subscriptions` is detected, an exception will be thrown to prevent potential loss of messages. 
+
+To move messages from the `NServiceBus.Subscriptions` queue to the new queue, refer to the instructions outlined below.
+
+## Create the subscriptions queue
+
+Create a transactional queue called [EndpointName].Subscriptions and substitute the actual name for the endpoint. This can be done by using the Powershell scripts, using [native tools or other tools like QueueExplorer](/transports/msmq/viewing-message-content-in-msmq.md#windows-native-tools). If creating queues manually using UI tools, ensure that the queues are marked, "transactional". 
```

Comment:
I expected something after "native tools", because of the comma. Should `EnableInstallers` not be mentioned here?

Reply:
@dvdstelt - I've changed it to 

>PowerShell scripts, Windows MMC Snap-in or other tools like QueueExplorer

Hope this is clearer now.